Extensions/Effects/reflection-pixi-filter.js

```diff
@@ -19,12 +19,16 @@ gdjs.PixiFiltersTools.registerFilterCreator("Reflection", {
       ],
       time
     );
-
+    reflectionFilter._animationTimer = 0;
     return reflectionFilter;
   },
   update: function(filter, layer) {
-    if (filter.animated) {
-      filter.time -= layer.getElapsedTime() / 1000;
+    if (filter.animationFrequency !== 0) { 
+      filter._animationTimer += layer.getElapsedTime() / 1000;
+      if (filter._animationTimer >= 1 / filter.animationFrequency) {
+        filter.time += layer.getElapsedTime() / 1000;
```

Comment:
Same, time must be continuously updated. Here you both use a timer and then update the time again with elapsedTime => this does not make sense in terms of time.

Frequency is only for seed.